example-config.yml

```diff
@@ -374,6 +374,28 @@ itinerary:
 #       - WALK
 #       - BICYCLE
 
+### Configure here the carbon output in grams per meter for each mode.
+### massUnits can be 'pound', 'kilogram', 'ounce', or 'gram'
+# co2:
```

Comment:
Are these still accurate? I'm not seeing them have an effect on the results. Additionally, the defaults seem a bit strange: for a certain itinerary I'm getting that walking is 85% of driving alone, but scooter is 100% less? That doesn't make any sense

Reply:
The mass units should show up in the trip details, where the actual mass of the CO2 is shown. It won't have any effect on the percentages, of course.

Reply:
Hm there does appear to be an issue where the units are not changing. I think this is a problem on the OTP-UI side, since the entire CO2 config just gets passed into TripDetails.

Reply:
And finally, can you give me an example itinerary where you're seeing the wacky numbers? That sounds weird.

Reply:
I'm seeing it on all e-scooter share itineraries. My guess is it may be related to the name changing between `SCOOTER` to `MICROMOBILITY_RENT`? In that case I think our fallback should be infinite carbon rather than 0 carbon

Reply:
I've now discovered this was a configuration issue. Thanks for the new documentation!